packages/rmf-codegen/scripts/helpers.js

```diff
@@ -0,0 +1,18 @@
+const shelljs = require('shelljs');
```

Comment:
shelljs is overkill here and not a dependency of the package.  Iit's intended to normalize common commands like copying / moving / listing / finding files across operating systems and distros, but for just calling a command it's not needed).